src/SW_Site.c

```diff
@@ -430,6 +430,7 @@ void SWRC_PTF_estimate_parameters(
 			LOGERROR,
 			"PTF is not implemented in SOILWAT2."
 		);
+        return; // Exit function prematurely due to error
 	}
 
 	/**********************************/

```

Comment:
Why did the function `SW_swcBulk_saturated()` not get a premature `return` in 
```
default:
			LogError(
				LogInfo,
				LOGERROR,
				"`SW_swcBulk_saturated()`: SWRC (type %d) is not implemented.",
				swrc_type
			);
			break;
```
?